sflvault/config/config.py

```diff
@@ -0,0 +1,36 @@
+import os
+import yaml
+
+
+class Config():
+
+    def __init__(self, path):
+
+        self.default_config_location = path
+
+        try:
+            with open(self.default_config_location, 'r') as f:
+                self.config = yaml.safe_load(f)
+        except IOError:
+            os.sys.exit("Error while reading configuration file")
```

Comment:
A `raise` here would be best, that way errors can be handled in a higher frame according to the context